examples/SerialRadio/SerialRadio.ino

```diff
@@ -76,15 +76,23 @@ int    i_sidx = 5;        ///< Start at Station with index=5
 /// by uncommenting the right radio object definition.
 
 // RADIO radio;       ///< Create an instance of a non functional radio.
-RDA5807M radio;    ///< Create an instance of a RDA5807 chip radio
-// SI4703   radio;    ///< Create an instance of a SI4703 chip radio.
+// RDA5807M radio;    ///< Create an instance of a RDA5807 chip radio
+SI4703   radio;    ///< Create an instance of a SI4703 chip radio.
 //SI4705   radio;    ///< Create an instance of a SI4705 chip radio.
 // TEA5767  radio;    ///< Create an instance of a TEA5767 chip radio.
 
 
 /// get a RDS parser
 RDSParser rds;
 
+/// Modified Julian Day from RDS Time
+typedef struct MJD_INFO {
+  int month;
+  int day;
+  int year;
+};
+

```

Comment:
The MDJ structures and functions should be moved into RDSParser to be available is all sketches.